tests/integration/targets/connection_aws_ssm/aws_ssm_integration_test_setup/tasks/main.yml

```diff
@@ -109,10 +109,11 @@
         tags:
           ansible-test: '{{ resource_prefix }}'
 
-    # - name: Create S3 bucket
-    #   s3_bucket:
-    #     name: "{{resource_prefix}}-aws-ssm-s3"
-    #   register: s3_output
+    - name: Create S3 bucket
```

Comment:
I reenabled commented task to create the bucket on demand

Reply:
Can you please replace `resource_prefix` with `tiny_prefix`. Otherwise we hit the 63 char limit time to time.